examples/seq2seq/seq2seq_trainer.py

```diff
@@ -62,9 +81,19 @@ def create_optimizer_and_scheduler(self, num_training_steps: int):
                 )
 
         if self.lr_scheduler is None:
```

Comment:
I know it could be set already, but I would rather just warn if it is not None and then overwrite no matter what?
Or am I missing something?

Reply:
why should  we overwrite ? `Trainer` allows you pass to `optimizer` and `scheduler` if you need something different from default, so I if passed I can think we shouldn't overwrite

Reply:
ok well then we are overwriting/ignoring the `--lr_scheduler` arg. Fine with me. But probably worth a comment/or warning.